e2e/provision/playbooks/roles/bootstrap/tasks/prep-gitea.yml

```
@@ -18,6 +18,27 @@
       metadata:
         name: "{{ gitea.k8s.namespace }}"
 
+- name: Create docker hub secret
+  when:
+    - DOCKERHUB_USERNAME is defined
+    - DOCKERHUB_TOKEN is defined
```

Shouldn't be better to read it from the environment variables?
```suggestion
    - lookup('ansible.builtin.env', 'DOCKERHUB_USERNAME', default=Undefined) is defined
    - lookup('ansible.builtin.env', 'DOCKERHUB_TOKEN', default=Undefined) is defined
```

---

Good suggestion

---

@electrocucaracha I commited your suggestion and it seems to be failing the e2e at http://prow.nephio.io/view/gs/prow-nephio-sig-release/pr-logs/pull/nephio-project_test-infra/182/e2e-fedora-34/1699803133871394816#1:build-log.txt%3A11586

---

The previous suggestion needs to be used in all the DOCKERHUB_ occurrences, in other words you have to also used in the line 27.

---

oh, I see, let me try that

---

@electrocucaracha seems like it did not work as can be seen at http://prow.nephio.io/view/gs/prow-nephio-sig-release/pr-logs/pull/nephio-project_test-infra/182/e2e-ubuntu-focal/1699898955883614208#1:build-log.txt%3A10476
for now, I have will revert the changes to what works fine in my environment and we shall continue to make fine tunings later.